package.json

```diff
@@ -120,6 +120,23 @@
             "default": true,
             "description": "%configuration.SAS.session.outputHtml%"
           },
+          "SAS.results.html.style": {
+            "order": 1,
+            "type": "string",
+            "default": "(auto)",
+            "enum": [
+              "(auto)",
+              "(server default)",
+              "HighContrast",
+              "Ignite",
+              "Illuminate"
+            ],
+            "enumDescriptions": [
+              "%configuration.SAS.results.html.style.(auto)%",
+              "%configuration.SAS.results.html.style.(server default)%"
+            ],
```

Comment:
We can also use "enumItemLabels" here if we want to provide translatable enum labels. It may be appropriate, at least for the "(auto)" and "(server default)" values? If we do that, it displays both the label and the enum value, so it would look something like this:
![image](https://github.com/sassoftware/vscode-sas-extension/assets/9735352/9c6a8722-b924-4e8e-bce0-7619f53029c7)
I noticed a lot of other options in VS Code use "auto", not in parens, and apparently not translated.

Reply:
Since "HighContrast" is the actual name of the style used in SAS code (as well as the other explicit styles listed), I would expect those to be translated.